open-company/open-company-web
# Mobile ro

card: https://trello.com/c/weIfykPI

**NB: merge after #78 since this branch is forked from big-web branch.**

To test:
- make sure you are logged in with slack
- select a company with write permissions
- resize you browser window to be at most 970px wide
- try to manage the topics, you can't? good
- try to make the edit button appears, you can't? good
- try to edit a topic in any other way you think you could, you can't? good
- merge
